analyzer/src/funTest/assets/projects/external/example-python-flask-expected-output.yml

```diff
@@ -33,26 +33,23 @@ packages:
 - id: "PyPI::click:8.1.3"
   purl: "pkg:pypi/click@8.1.3"
   authors:
-  - "Armin Ronacher"
+  - "Armin Ronacher <armin.ronacher@active-4.com>"
```

Comment:
commit-msg: "move **to** the strongest algorithm that is available."

Note: switching to a different hash algorithm could be factored out into a separate commit.

Reply:
I don't want to spend the time on updating all the expected result files again twice.

Reply:
Commit message fixed.